pkg/loop/config.go

```diff
@@ -0,0 +1,117 @@
+package loop
+
+import (
+	"fmt"
+	"net/url"
+	"os"
+	"strconv"
+	"strings"
+)
+
+const (
+	envPromPort = "CL_PROMETHEUS_PORT"
+	//TODO CL_ prefix everything?
```

Comment:
WDYT? @krehermann @patrickhuie19

Reply:
What's the problem we're solving? Distinguishing between the environment of the host and plugins?

Reply:
Namespacing so we don't have collisions. We do this in core. These aren't user facing so it's not as important, but in theory we could have a conflict with a dependency that has it's own env vars :shrug:

Reply:
Hmm - If we prefix everything with `CL_`, then we're still at risk of collisions.

Reply:
Indeed, but they are less likely :shrug: